src/pages/[marketplace]/deals/index.tsx

```diff
@@ -147,7 +157,7 @@ const Deals: NextPageWithLayout = () => {
 									},
 								};
 							}}
-							dataSource={pendingDeals?.map((deal) => mapDeal(deal))}
+							dataSource={pendingDeals && mapDeals(pendingDeals)}
 							columns={dealsTableColumns}
 						/>
 					</TabPane>
```

Comment:
Is there a reason for not sorting the ended deals below?

Reply:
nope :p